docs/source/conf.py

```diff
@@ -235,6 +235,13 @@
 
 
 # -- Extension configuration -------------------------------------------------
+extensions = [
+    "sphinx_github_changelog",
+]
+sphinx_github_changelog_token = os.environ.get("CHANGELOG_GITHUB_TOKEN")
```

Comment:
in my local environment, i have no issues building without this, while having the environment variable in place.
The issue is not in the configuration of this project, but that the server that is building the spec (whichever that may be) does  not have the token set.

Reply:
@christian-herber-nxp I thought that this is how the doc server gets the token set?

Reply:
With my rights, I do not have visibility on this. As the regular builds on RTD are working, my assumption is that there is a different configuration for the github PR triggered builds. This is a server configuration issue, and nothing to do with the project as it is.